ext/web/lib.rs

```diff
@@ -146,21 +149,41 @@ pub fn init<P: TimersPermission + 'static>(
 }
 
 fn op_base64_decode(
-  _state: &mut OpState,
+  _: &mut OpState,
   input: String,
   _: (),
 ) -> Result<ZeroCopyBuf, AnyError> {
-  let mut input: &str = &input.replace(|c| char::is_ascii_whitespace(&c), "");
+  Ok(b64_decode(input, false)?.into())
+}
+
+fn op_base64_atob(
+  _: &mut OpState,
+  input: String,
+  _: (),
+) -> Result<ByteString, AnyError> {
+  Ok(ByteString(b64_decode(input, true)?))
+}
+
+fn b64_decode(input: String, padding: bool) -> Result<Vec<u8>, AnyError> {
+  let mut input = input.into_bytes();
```

Comment:
How about using a byte string as input? Have you checked if that would be faster?

Reply:
Yeah was thinking that too, just added it and it's faster bringing us down from ~7ms to ~6ms in that bench